test/common.js

```diff
@@ -497,6 +497,12 @@ function fail(msg) {
 }
 exports.fail = fail;
 
+exports.mustNotCall = function(msg) {
+  return function dontCall() {
+    fail(msg || 'function should not have been called');
```

Comment:
On second thought I think this is not needed as the function must be invoked explicitly and using a non-string argument probably doesn't make sense.